docs/core/components/start-form.component.md

```diff
@@ -28,8 +28,9 @@ Displays the Start [`Form`](../../../lib/process-services/task-list/models/form.
 | ---- | ---- | ------------- | ----------- |
 | data | [`FormValues`](../../../lib/core/form/components/widgets/core/form-values.ts) |  | Custom form values map to be used with the rendered form. |
 | disableCompleteButton | `boolean` | false | If true then the `Complete` outcome button is shown but it will be disabled. |
+| disableSaveButton | `boolean` | false | If true then the `Save` outcome button is shown but it will be disabled. |
```

Comment:
If true then the `Save` outcome button is shown but it will be disabled. => If true then the `Save` outcome button is shown but will be disabled.